components/logo/README.md

```diff
@@ -4,18 +4,67 @@
 
 ## Toegankelijkheid
 
```

Comment:
Misschien kunnen we dit toevoegen:

- Als je inspraak hebt in het ontwerpproces van je logo, probeer dan een minimale contrastverhouding van 4,5:1 aan te houden (of 3:1 voor grotere tekst).
- Besteed aandacht aan alle logo's op de website, niet alleen aan het bedrijfslogo